### PR TITLE
Updated template to produce a Python 3.5 compatible project

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-2843

### CODE REVIEW

**Comment by clintonb** [comment]
@cpennington please review

FYI @nedbat @edx/ecommerce @edx/devops

**Comment by rlucioni** [comment]
Nice. 👍

File: {{cookiecutter.repo_name}}/requirements/production.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Packages required in a production environment
 -r base.txt
 
-gevent==1.0.2
-gunicorn==19.3.0
-MySQL-python==1.2.5
-PyYAML==3.11
+gevent
+gunicorn
```
Comment (cpennington) [inline {{cookiecutter.repo_name}}/requirements/production.txt:5]: Any minimum requirements on gevent and gunicorn to be py3.5 compatible?

Comment (clintonb) [inline {{cookiecutter.repo_name}}/requirements/production.txt:5]: At this point the latest version of either library is Python 3.5 compatible. I'd be really surprised if developers chose anything besides a recent version.

**Pull request merged**